src/main/java/org/opensrp/repository/postgres/mapper/custom/xml/CustomOrganizationMapper.xml

```diff
@@ -96,6 +96,48 @@
       </foreach>
     </where>
   </sql>
-
+	<resultMap id="resultMap" type="org.opensrp.domain.Organization">    
+	    <id column="id" jdbcType="BIGINT" property="id" />
+	    <result column="identifier" jdbcType="VARCHAR" property="identifier" />
+	    <result column="active" jdbcType="BIT" property="active" />
+	    <result column="name" jdbcType="VARCHAR" property="name" />    
+	    <result column="date_deleted" jdbcType="TIMESTAMP" property="dateDeleted" />
+	    <result column="parent_id" jdbcType="BIGINT" property="partOf" />
+	    <result column="member_count" jdbcType="BIGINT" property="memberCount" />
+	    
+  	</resultMap>
+  	<sql id="query">
+		select o.id,o.identifier,o."name",o."type",o.parent_id,count(distinct(pr.id))as member_count 
+		from team.organization as o
+		left join team.practitioner_role as pr on o.id = pr.organization_id
+		left join team.organization_location  ol on ol.organization_id =o.id
+		<where> 
+			o.date_deleted is null	
+			<if test="searchBean.name != null">
+			<bind name="name" value="'%' + searchBean.name + '%'" />
+				and o."name" ilike #{name} 
+			</if>
+			<if test="searchBean.locations != null">
+				and ol.location_id in
+				<foreach close=")" collection="searchBean.locations" item="listItem" open="(" separator=",">
+					#{listItem}
+				</foreach>
+			</if>
+		</where>
+		group by o.id,o.identifier,o."name",o."type",o.parent_id
+	</sql>
+	<select id="selectSearchOrganizations" resultMap="resultMap">
+	
+		<include refid="query"/>	
+		<if test="searchBean.orderByFieldName!=null and searchBean.orderByType!=null">
+			order by ${searchBean.orderByFieldName}  ${searchBean.orderByType}  
+		</if>
+		LIMIT #{limit} OFFSET #{offset}	
+	</select>
+	<select id="selectOrganizationCount" resultMap="resultMap">	
```

Comment:
change this to select the `count(distinct o.id) `and the return type to int

This would imply removing the below from `<sql id="query">` and adding them to `selectSearchOrganizations` 
```
select o.id,o.identifier,o."name",o."type",o.parent_id,count(distinct(pr.id))as member_count 
group by o.id,o.identifier,o."name",o."type",o.parent_id
```